lightning/src/ln/channelmanager.rs

```diff
@@ -2873,9 +2872,11 @@ const MAX_NO_CHANNEL_PEERS: usize = 250;
 /// short-lived, while anything with a greater expiration is considered long-lived.
```

Comment:
Doesn't look like `MAX_SHORT_LIVED_RELATIVE_EXPIRY` is used anywhere outside of tests now?

Reply:
Thanks for catching it!
Updated it to be test only `const` in [pr3246.18](https://github.com/shaavan/rust-lightning/commits/pr3246.18)